users/README.md

```diff
@@ -61,6 +62,29 @@ Returns all users in the system.
   - **Code:** 401
     - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
+## **GET /users/self**
+
+Returns the details of user.
```

Comment:
Which user?
`Returns the details of the logged in user` makes more sense to me.

Reply:
Sure